src/lib/db/createStore.ts

```diff
@@ -922,6 +928,7 @@ export function createDbStoreCreator(
               return {
                 ...(await targetStorage.db.getTag(tagName))!,
                 name: tagName,
+                color: '',
```

Comment:
If color is not set, I'd like to leave it as  `undefined`

Reply:
Removed!